### ISSUE TITLE
Textual description missing for interactive image

### ISSUE THREAD

**Issue by michellenSD** [logicly-au/docs.iar-dst.online#29]
**'Child Domains Overview' page, Steps to Generate PDF report:**

Textual description missing for interactive image:
* Hamburger menu 
<img width="95" alt="Image" src="https://github.com/user-attachments/assets/3fab18b7-14b8-4161-98c4-f667719807a0" />

Textual description not announced for image button:
* The expand /collapse image buttons under "Table of content" heading
<img width="225" alt="Image" src="https://github.com/user-attachments/assets/1bd6b2c9-90aa-45ee-9d04-a2f44b113377" />

_Accessibility report issues 38, 40_

See https://gh.sdintra.net/IAR-DST/iar-dst-app/issues/288 for report
